testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/theme/ThemeResourceProviderTest.java

```diff
@@ -50,4 +51,18 @@ public void getResourceAsStream() {
         });
     }
 
+    @Test
+    public void getMessages() {
+        testingClient.server().run(session -> {
+            try {
+                ThemeProvider extending = session.getProvider(ThemeProvider.class, "extending");
+                Theme theme = extending.getTheme("base", Theme.Type.LOGIN);
+                System.out.println(theme.getMessages("messages", Locale.ENGLISH));
+                Assert.assertNotNull(theme.getMessages("messages", Locale.ENGLISH).get("test.keycloak-8818"));
+                Assert.assertEquals("Keycloak Admin Console (from theme-resources)", theme.getMessages("admin-messages", Locale.ENGLISH).get("consoleTitle"));
```

Comment:
Overriding messages should not be possible with a theme resource so this should return the original message.

Reply:
Please keep the attempt at overriding the message though so we know themes override theme resources and not the other way around

Reply:
done, I had to use another key as the bundle admin-messages is not in the LOGIN theme so the test was bad anyway.